### PR TITLE
docs: update beta delete samples for video

### PR DESCRIPTION
**Pull request by nnegrey**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/nodejs-automl/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

File: samples/test/delete-dataset.beta.test.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    };
+    const [response] = await client.createDataset(request);
+    datasetId = response.name
+      .split('/')
+      [response.name.split('/').length - 1].split('\n')[0];
+  });
+
+  it('should delete a dataset', async () => {
+    const projectId = await client.getProjectId();
+    const delete_output = execSync(
```
Comment (JustinBeckwith) [inline samples/test/delete-dataset.beta.test.js]: Generally prefer camelCase to snake_case in JavaScript :)

File: samples/test/delete-model.beta.test.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    // nonexistent model and confirm that the model was not found, but other
+    // elements of the request were valid.
+    const projectId = await client.getProjectId();
+    const args = [
+      DELETE_MODEL_REGION_TAG,
+      projectId,
+      LOCATION,
+      'TRL0000000000000000000',
+    ];
+    const output = cp.spawnSync('node', args, {encoding: 'utf8'});
```
Comment (JustinBeckwith) [inline samples/test/delete-model.beta.test.js]: You probably want to copy/paste the `execSync` code to here and use that as well.  I don't think `spawnSync` returns the output in quite this way (could be wrong).  Regardless, would love to use `execSync` for consistency.

Comment (nnegrey) [reply samples/test/delete-model.beta.test.js]: I had to switch to `spawnSync` for the methods where I need to catch the error from the sample. 
`execSync` seems to only throw the error and the error is what I want to check in this case.

**Comment by nnegrey** [comment]
Before I merge, I'll try to see if I can re-introduce any flakiness that might exists.

**Comment by nnegrey** [comment]
Ran the tests several times back to back, no issues. I think it was related to the batch tests fixed in a different PR

**Pull request merged**